src/utils/destroy-backbone-view.js

```diff
@@ -11,6 +12,7 @@ export default function destroyBackboneView(view) {
     triggerMethodOn(view, 'before:detach', view);
   }
 
+  dom.removeEl(view.el);
```

Comment:
Just a note.  Doesn't particularly matter now that it's changed, but `$.remove` also unbinds events.. so for a vanilla view (or for when we override remove) `this.undelegateevents` will be necessary.